package.json

```
@@ -9,8 +9,8 @@
     "generate:typechain": "typechain --target ethers-v6 --out-dir generated/typechain './node_modules/@iexec/poco/build/contracts/*.json' --ts-nocheck",
     "build": "rm -rf generated && rm -rf build && npm run generate:typechain && graph codegen && graph build --network ${NETWORK_NAME:-bellecour}",
     "test": "npm run test:unit && npm run test:e2e",
-    "test:unit": "graph test",
-    "test:e2e": "NETWORK_NAME=${NETWORK_NAME:-bellecour} mocha",
+    "test:unit": "graph test unit",
+    "test:e2e": "mocha",
```

what do you think of doing 
```suggestion
    "test:e2e": "mocha tests/e2e/**/*.ts",
```
and removing line 3 from `mocharc.json`?
It seems more intuitive to me.

---

fix Commit: 814adf641141aa766b3495d8e8d59e095444ce36